frontend/src/router.js

```diff
@@ -262,6 +262,21 @@ export default new Router({
         }),
       },
     },
+    {
+      name: 'material/lists',
+      path: '/camps/:campId/:campTitle?/material/lists',
+      components: {
+        navigation: NavigationCamp,
+        default: () => import('./views/material/MaterialLists.vue'),
+      },
+      beforeEnter: all([requireAuth, requireCamp]),
+      props: {
+        navigation: (route) => ({ camp: campFromRoute(route) }),
+        default: (route) => ({
+          camp: campFromRoute(route),
+        }),
+      },
+    },
```

Comment:
I think `material/lists` should move to the camp route object as a child. (But not `material/all` as it needs the sidebar.